tools/packaging/kernel/configs/fragments/x86_64/sgx.conf

```diff
@@ -1 +1,4 @@
+# Intel Software Guard Extensions
+
 CONFIG_X86_SGX=y
+CONFIG_NUMA=y
```

Comment:
@cmaf Do you know why this option is required? Also can you add a comment here that it is specifically for Gramine.

Reply:
@amshinde it is specified in our own documentation in a [note](https://github.com/kata-containers/kata-containers/blob/main/docs/use-cases/using-Intel-SGX-and-kata.md#notes) for using kata containers with Gramine:

https://github.com/kata-containers/kata-containers/blob/6ffdebd202223407f0306ea346ba777836923465/docs/use-cases/using-Intel-SGX-and-kata.md?plain=1#L100-L102

I can't find a reference in it in the Gramine docs and don't know offhand why it is required, so maybe @mythi or @jodh-intel could shed some light on whether or not it is still a requirement. (Referring to changes in 8c8bcb7 https://github.com/kata-containers/kata-containers/pull/2810)

Reply:
Gramine wants to verify the CPU/NUMA topology etc. Without this config option, `/sys/devices/system/node/*` etc are not available for containers to read and gramine fails because of that.